Sources/Relays/CurrentValueRelay.swift

```diff
@@ -46,6 +46,10 @@ public class CurrentValueRelay<Output>: Relay {
         publisher.subscribe(storage)
     }
 
+    public func subscribe<P: Relay>(_ publisher: P) -> AnyCancellable where Output == P.Output {
+        publisher.subscribe(storage)
```

Comment:
Thanks! I'm wondering what's the difference between this and `subscribe(_ publisher:)` ? It seems to be basically identical. Why does this solve the issue?

Reply:
@freak4pc Honestly, I'm not quite sure because this is something I found experimentally.
It might be somehow connected to the fact that Relays do not conform to the Subject protocol, but I haven't found anything that could confirm it yet.

Reply:
Alright, I think I'm OK with aligning it this way. Thanks!